rllib/policy/policy.py

```diff
@@ -189,6 +191,39 @@ def compute_single_action(self,
         return single_action, [s[0] for s in state_out], \
             {k: v[0] for k, v in info.items()}
 
+    @ExperimentalAPI
+    def compute_actions_from_trajectories(
```

Comment:
Can we omit experimental? I think a comment is sufficient since thru are temporary and rare.